### ISSUE TITLE
Question: What about node.js

### ISSUE THREAD

**Issue by VMF214** [GeniusesOfSymfony/WebSocketBundle#186]
have you got an idea or an exemple script for node.js?
Thanks in advance!

**Comment by VMF214** [comment]
i try this but it doesn't work...
it log this : 

unreachable
{ reason: null,
  message: null,
  retry_delay: 1.6512635605121795,
  retry_count: 1,
  will_retry: true }



`var autobahn = require('autobahn');

var connection = new autobahn.Connection({url: 'ws://mydomain:8080', realm: false});
connection.onopen = function (session) {
   function onevent(args) {
      console.log("Event:", args[0]);
   }
};
connection.onclose = function(error,reason){
  console.log(error);
  console.log(reason);
};
connection.open();
`




the following script work (in browser):



    var webSocket = WS.connect("ws://mydomain:8080");
    webSocket.on("socket/connect", function(session){
        //session is an Autobahn JS WAMP session.
        console.log("Successfully Connected!");
        //the callback function in "subscribe" is called everytime an event is published in that channel.
        session.subscribe("arduino/channel", function(uri, payload){
            console.log(payload);
            $('#channel').prepend('<p>' + payload.date + ':&nbsp;&nbsp;' + payload.content +':&nbsp;&nbsp;' +payload.ip +':&nbsp;&nbsp;' + payload.display +'</p>');
        });
    });
    webSocket.on("socket/disconnect", function(error){
        //error provides us with some insight into the disconnection: error.reason and error.code
        $('#channel').prepend('<p>' + "Disconnected for " + error.reason + " with code " + error.code +'</p>');
    });

**Comment by CoalaJoe** [comment]
This Bundle runs on Symfony 2 not node.js

**Comment by VMF214** [comment]
yes i know, and it work well, but my goal is to use a node.js client on client side, not on html browser.

**Comment by ProPheT777** [comment]
> node.js client on client side

You can use any client library compliant with WAMP protocol v1 in any language. (like this for example : https://www.npmjs.com/package/wamp1)
